dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/adaptor/ESAAdaptor.java

```diff
@@ -204,13 +204,15 @@ private static void install(Product product, ESAAsset featureAsset, Provisioning
                                     featureDefinition, fr, processedEntries, checksumInput, checksumOutput, executableFiles, extattrFiles, checksumsManager);
 
                 }
-            } else if (SubsystemContentType.FEATURE_TYPE == type && !skipFeatureDependencyCheck) {
-                // Look to see if the feature already exists and try to find it in the same place as the current ESA if we don't have it
-                String symName = fr.getSymbolicName();
-                if (!product.containsFeature(symName) && !product.containsFeatureCollection(symName) && !featuresToBeInstalled.contains(symName)) {
-                    throw new InstallException(Messages.PROVISIONER_MESSAGES.getLogMessage("tool.install.missing.feature", featureDefinition.getSymbolicName(), symName));
+            } else if (SubsystemContentType.FEATURE_TYPE == type) {
```

Comment:
Is the new condition same as the original one?

            } else if (SubsystemContentType.FEATURE_TYPE == type) {
                if (!skipFeatureDependencyCheck) {

and

} else if (SubsystemContentType.FEATURE_TYPE == type && !skipFeatureDependencyCheck) {